index.html

```diff
@@ -272,5 +272,73 @@ <h2>About me</h2>
         </div>
       </div>
     </main>
+    <footer class="footer">
+      <form
+        action="https://formspree.io/f/contact-form"
+        method="post"
+        class="contact-form"
+      >
+        <h3>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h3>
+        <ul>
+          <li>
+            <input
+              type="text"
+              placeholder="Full name"
+              required
+              maxlength="30"
+            />
+          </li>
+          <li>
+            <input type="email" placeholder="Email address" required />
+          </li>
+          <li>
+            <textarea name="message" cols="30" rows="10" maxlength="500">
+Enter text here</textarea
```

Comment:
- [ ] Great job :+1: :100: with `textarea`. But it should be better for your `Enter text here` text to be in the `Placeholder` property. So this will not affect the user when he wants to write a message.